### PR TITLE
7 covid symptoms dictionary is missing medical synonyms like sob shortness of breath

### PR DESCRIPTION
**Pull request by comorbidity**
updated COVID symptoms dictionary with Doctors input (Amy and Alon)

### CODE REVIEW

File: test/resources/covid_symptoms.bsv [hunk 1]
```diff
@@ -1,26 +1,134 @@
 #CUI|TUI|CODE|SAB|STR|PREF
```
Comment (mikix) [inline test/resources/covid_symptoms.bsv:1]: I think this file got moved a while ago into `ctakesclient/resources/` as part of exposing it in the API to cumulus.

Also, I think your unit tests might be failing because the bsv loader doesn't like empty lines?

**Review by mikix — APPROVED** [review]
Conceptually fine and looks good - if tests & linters pass, I'm happy.

**Review by mikix — APPROVED** [review]
Nice. I like re-enabling the SOB test too - good catch
